db/schema.rb

```diff
@@ -31,6 +31,7 @@
     t.integer "selection_type", default: 0, null: false
     t.integer "min_players_per_team"
     t.integer "max_players_per_team"
+    t.text "image_data"
```

Comment:
Where does this come from now? :O

Reply:
Great review btw @MariusDanner :D

Reply:
I had to run the migrations again. The schema.rb seems to have been outdated ;) That's fine ;)